### ISSUE TITLE
Newbie with Arlo Q cameras that do not show up in devices

### ISSUE THREAD

**Issue by KjacobsenDF** [twrecked/hass-aarlo#358]
Hi there, 
First, I want to say thank you for creating this package. I have had Arlo Q cameras for years, and the geofencing has never worked properly from the app. I had been using IFTTT with Life360, which worked great, but IFTTT is no longer playing nicely with Life360. Sigh. I waded into HA in the hopes that I could get the same functionality, albeit with more work up front. I installed HA on a raspberry pi 4, and I believe I successfully installed aarlo. (At least when I go into HACS, I see "Arlo Camera Support" as an integration, so I think that means it worked?) But what I cannot seem to figure out is how to get the cameras themselves to show up in the devices. I set up a separate Arlo account and granted access to the cameras to it, and I put that username/password combination into the configuration.yaml file. But I cannot see any cameras at all. I've restarted and rebooted the R-pi, but to no avail. I have no idea what I am doing and I cannot seem to find any info/suggestions that work. I feel like I am SO CLOSE to making this work and I do not want to give up now. Thanks for any help/suggestions you might have. Sorry if I am being totally stupid.

**Comment by twrecked** [comment]
Did you add `aarlo` to the camera config? [see here](https://github.com/twrecked/hass-aarlo#camera-configuration)

**Comment by KjacobsenDF** [comment]
I did.  My configuration file has these entries:

aarlo:
  username: [username]
  password: [password]
  refresh_devices_every: 2
  stream_timeout: 120

camera:
  - platform: aarlo

sensor:
  - platform: aarlo
    monitored_conditions:
    - total_cameras
    - last_capture
    - recent_activity
    - captured_today
    - signal_strength

Thanks for your help!

**Comment by twrecked** [comment]
That's everything you need. Your cameras should be there. 

First, did you give your second account admin access? That might affect the Q cameras.

Are they there but not being displayed? Look under `Developer Tools [the hammer icon]` --> `STATES` and look for any `camera.aarlo_` entries. Look for any `aarlo` entries.

**Comment by KjacobsenDF** [comment]
Ah ha, admin access was the key (and also disabling 2FA via text)!  They are now there in developer tools, but they're still not being displayed on the devices page.  BUT this gives me hope that by messing around with it more, I will get everything working properly after all!  Thanks for your help!

**Comment by twrecked** [comment]
The Arlo Qs are their own base station so you need the extra permissions.

I'm not sure what you mean by devices page - but you can show them on the front end using the [lovelace Arlo] card. It can be installed from HACS as well.

**Comment by KjacobsenDF** [comment]
Thank you!  I can see them on the "overview" page now.  I just set up the automations I need, and they appear to be working.  (Well, at least as I sit here, if I arm the cameras via the Arlo app and then execute the automation to disable motion detection when I am home, I see them go from armed to disarmed in the app, so this makes me think the one I made to arm the cameras when I leave home will work tomorrow when I actually leave the house.)  THANK YOU for writing all of the code to make the Arlos controllable, and THANK YOU for helping me make this work.
